contracts/RegistryAdapters/IronBankRegistryAdapter.sol

```diff
@@ -583,18 +633,23 @@ contract RegistryAdapterIronBank is Ownable {
         IUnitroller.Market memory market =
             IUnitroller(comptrollerAddress).markets(assetAddress);
 
-        uint256 balance = assetBalance(assetAddress);
         TokenAmount memory underlyingTokenBalance =
-            tokenAmount(balance, tokenAddress, tokenPriceUsdc);
+            tokenAmount(assetBalance(assetAddress), tokenAddress, tokenPriceUsdc);
 
-        uint256 totalBorrowed = asset.totalBorrows();
         uint256 totalBorrowedUsdc =
             IOracle(oracleAddress).getNormalizedValueUsdc(
                 tokenAddress,
-                totalBorrowed,
+                asset.totalBorrows(),
```

Comment:
Similarly is this change made due to stack depth issues?

Reply:
Yes, also stack too deep issue